src/constants.js

```diff
@@ -19,7 +19,7 @@ export const EthereumNetworkKeys = Object.freeze({
 export const SubstrateNetworkKeys = Object.freeze({
   KUSAMA: '456',
```

Comment:
looks like it needs to merge master?

Reply:
I messed up the merge, I had it merged locally.